rspec-buildkite-analytics.gemspec

```diff
@@ -24,7 +24,7 @@ Gem::Specification.new do |spec|
 
   spec.required_ruby_version = Gem::Requirement.new(">= 2.3.0")
 
-  spec.add_dependency "activesupport", ">= 5.2", "<= 7.0"
+  spec.add_dependency "activesupport", ">= 5.2", "< 8"
```

Comment:
`< 7.1` would be fine too, if being extra cautious about minor releases is preferred 🤷